data.sql

```diff
@@ -1,6 +1,6 @@
 /* Populate database with sample data. */
```

Comment:
- [ ] It would be nice if you make your branch name descriptive such as `feature/update_animals` because I think creating animals was the first milestone 🤔 Kindly make your branch descriptive 👍